### ISSUE TITLE
`mathtools` causes "TeX capacity exceeded" error

### ISSUE THREAD

**Issue by ajmalanoski** [andrasbarany/cascadilla-xelatex-biblatex#2]
Loading the `mathtools` package gives the following error: "TeX capacity exceeded, sorry [parameter stack size=10000]."

The relevant parts of my preamble are as follows:

```
\documentclass[usegb4e]{cascadilla-xelatex-biblatex}

\usepackage{authblk}
\usepackage{mathtools}
  \DeclarePairedDelimiter\set\{\}
  \DeclarePairedDelimiter\tuple\langle\rangle
\usepackage{multicol}
\usepackage{multirow}
\usepackage{soul}
```

I've tried commenting out the other packages, but that didn't make the error go away. I've also used the same packages together with LaTeX's "article" document class without issue. Finally, I tried commenting out the `\DeclarePairedDelimiter` commands, and that didn't help either. It seems to be an interaction between `mathtools` and your document class.

**Comment by andrasbarany** [comment]
Thanks! For me, it appears to be a clash with the `usegb4e` option and the timing of loading `mathtools`. If you load mathtools in `cascadilla-xelatex-biblatex.cls`, do you still get the error message?

**Comment by ajmalanoski** [comment]
Yes, unfortunately I still get the error message. There's a possibility I'm doing it wrong though -- I haven't worked much with `.cls` files, so I'm not sure if I'm using the right commands or putting them in the right spot. I tried adding `usepackage{mathtools}` and `RequirePackage{mathtools}` before (I tried these one at a time, not both at the same time) before the lines where it looked like `gb4e` was being loaded:

```
\ifthenelse{\boolean{usegb4e}}{%
	\RequirePackage{gb4e}
	\def\glt{}
	\def\trans{}
```

But I got the same error when I tried to recompile my document.
